api/src/oscc.c

```diff
@@ -361,14 +438,15 @@ oscc_result_t oscc_disable_steering( void )
     return result;
 }
 
-void oscc_update_status( )
+
+void oscc_update_status( int sig, siginfo_t *siginfo, void *context )
```

Comment:
I did not know that it's possible to provide a context to a signal handler - this is _excellent_!